.github/workflows/build-lint-test.yml

```diff
@@ -7,6 +7,7 @@ on:
   push:
     branches:
       - main
+  merge_group:
```

Comment:
looking at the line `28` of this file where we have `deploy` command:

I'd assume (though not 100% sure) that now we'll deploy twice as this workflow is triggered first time on `merge_group` event and the second time on `pull_request`.

[Ref:](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue#triggering-merge-group-checks-with-github-actions.)
> The merge_group event is separate from the pull_request and push events.

Reply:
the workflow won't deploy if it's not triggered by a `pull_request` event https://github.com/cloudscape-design/actions/blob/e4833b7478b148dfbd3a58a4b3d90bc54a555d8c/.github/workflows/deploy.yml#L22